examples/lighting-app/k32w/main/AppTask.cpp

```diff
@@ -0,0 +1,561 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    Copyright (c) 2021 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include "AppTask.h"
+#include "AppEvent.h"
+#include "Server.h"
+#include "support/ErrorStr.h"
+
+#include "QRCodeUtil.h"
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/internal/DeviceNetworkInfo.h>
+
+#include "attribute-storage.h"
+#include "gen/attribute-id.h"
+#include "gen/attribute-type.h"
+#include "gen/cluster-id.h"
+
+#include "Keyboard.h"
+#include "LED.h"
+#include "LEDWidget.h"
+#include "TimersManager.h"
+#include "app_config.h"
+
+#define FACTORY_RESET_TRIGGER_TIMEOUT 6000
```

Comment:
nit: consider C++ style `constexpr int kFactoryResetTriggerTimeout = 6000` and similar.

Reply:
Fixed.